grails-app/views/occurrence/list.gsp

```diff
@@ -69,7 +69,11 @@
 ' Image does not support the identification of the species, subject is unclear and identifying features are difficult to see or not visible.<br/></div>',
             savePreferredSpeciesListUrl: "${createLink(controller: 'imageClient', action: 'saveImageToSpeciesList')}",
             getPreferredSpeciesListUrl:  "${createLink(controller: 'imageClient', action: 'getPreferredSpeciesImageList')}",
-            excludeCountUrl: "${createLink(controller: 'occurrence', action: 'dataQualityExcludeCounts', params: params.clone()).encodeAsJavaScript()}"
+            excludeCountUrl: "${createLink(controller: 'occurrence', action: 'dataQualityExcludeCounts', params: params.clone()).encodeAsJavaScript()}",
+            expandProfileDetails: ${expandProfileDetails},
+            userId: "${userId}",
+            prefKey: "${grailsApplication.config.dataquality.prefkey}",
```

Comment:
prefer `grailsApplication.config.getProperty("dataquality.prefkey", String, "")` to be used for these calls.